File: Software-Technologies/SoftUni Babies/CSharp/SoftUniBabies/Models/Baby.cs
namespace SoftUniBabies.Models
{
    using System;
    using System.ComponentModel.DataAnnotations;

    public class Baby
    {
        [Key]
        public int Id { get; set; }

        [Required]
        public string Name { get; set; }
        
        [Required]
        public string Gender { get; set; }

        public string Birthday { get; set; }
    }
}

File: Software-Technologies/SoftUni Babies/JavaScript/models/baby.js
const Sequelize = require('sequelize');

module.exports = function (sequelize) {
    let Baby = sequelize.define('Baby', {
        name: {
            type: Sequelize.TEXT,
            allowNull: false
        },
        gender: {
            type: Sequelize.TEXT,
            allowNull: false
        },
        birthday: {
            type: Sequelize.STRING,
            allowNull: false
        }
    });

    return Baby;
};